Doc/library/argparse.rst

```diff
@@ -1288,17 +1289,17 @@ this API may be passed as the ``action`` parameter to
    Action instances should be callable, so subclasses must override the
    ``__call__`` method, which should accept four parameters:
 
-   * ``parser`` - The ArgumentParser object which contains this action.
+   * parser - The ArgumentParser object which contains this action.
```

Comment:
```suggestion
   * *parser* - The ArgumentParser object which contains this action.
```
Parameters are conventionally formatted this way (same for the next few ones)

Reply:
Got it - this doesn't apply for parameters that are also jump targets right? Or is there a way to wrap something like `prog_` in * ?

Reply:
I think I remember someone finding a way to do that but it's very weird and probably not worth it.